int8 drone_id
crazyflie_trajectory_manager/ConfigTrajectoryStringPayload payload
---
bool proxy_success
bool result